nototools/android_patches.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/python
+# -*- coding: UTF-8 -*-
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Patches for Android versions of Noto fonts."""
+
+import glob
+from os import path
+
+from nototools import subset
+from nototools import coverage
+from nototools import fix_khmer_and_lao_coverage as merger
+from nototools import font_data
+from nototools import tool_utils
+from nototools import unicode_data
+
+from fontTools import ttLib
+from fontTools.ttLib.tables import otTables
+
+SRC_DIR = tool_utils.resolve_path('[tools]/packages/android')
+DST_DIR = tool_utils.resolve_path('[tools]/packages/android-patched')
+
+def _patch_hyphen_armenian_ethiopic():
```

Comment:
Yeah, I know.  Where is the code that defines what scripts these are?  Is
it in minikin somewhere?

On Tue, Mar 22, 2016 at 3:02 PM, jungshik <notifications@github.com> wrote:

> In nototools/android_patches.py
> <https://github.com/googlei18n/nototools/pull/204#discussion_r57077465>:
>
> > +from os import path
> > +
> > +from nototools import subset
> > +from nototools import coverage
> > +from nototools import fix_khmer_and_lao_coverage as merger
> > +from nototools import font_data
> > +from nototools import tool_utils
> > +from nototools import unicode_data
> > +
> > +from fontTools import ttLib
> > +from fontTools.ttLib.tables import otTables
> > +
> > +SRC_DIR = tool_utils.resolve_path('[tools]/packages/android')
> > +DST_DIR = tool_utils.resolve_path('[tools]/packages/android-patched')
> > +
> > +def _patch_hyphen_armenian_ethiopic():
>
> This need to be expanded to other script fonts that need to have a hyphen.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/googlei18n/nototools/pull/204/files/91e448dd04a0597555c280ff1d3a572ea3fdd96d#r57077465>
>